core/os/thread.cpp

```diff
@@ -28,6 +28,10 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
+// If PLATFORM_THREAD_OVERRIDE is defined, it will allow the platform to
+//    define its own thread implementation
```

Comment:
```suggestion
// If PLATFORM_THREAD_OVERRIDE is defined, it allows a given platform port
// to define its own thread implementation.
```

Reply:
Actually I guess you can remove this comment, having it once in the header is enough.